src/applications/ivc-champva/10-7959C/config/conditionalPaths.js

```diff
@@ -40,3 +40,11 @@ export function hasPrimaryProvider(formData, index) {
     'yes'
   );
 }
+
+export function hasSecondaryProvider(formData, index) {
+  if (index === undefined) return true;
+  return (
+    get('applicantHasSecondary.hasSecondary', formData?.applicants?.[index]) ===
+    'yes'
+  );
+}
```

Comment:
This `depends` function will get used for all pages in the secondary health insurance flow